pytest_remotedata/disable_internet.py

```diff
@@ -22,14 +24,31 @@
 _orig_opener = None
 
 
+def _resolve_host_ips(hostname, port=80):
+    """
+    Obtain all the IPs, including aliases, in a way that supports
+    IPv4/v6 dual stack.
+    """
+    try:
+        ips = set([s[-1][0] for s in socket.getaddrinfo(hostname, port)])
+    except socket.gaierror:
+        ips = set([])
+
+    ips.add(hostname)
+    return ips
+
+
 # ::1 is apparently another valid name for localhost?
 # it is returned by getaddrinfo when that function is given localhost
 
-def check_internet_off(original_function, allow_astropy_data=False):
+def check_internet_off(original_function, allow_astropy_data=False,
+                       allow_github_data=False):
     """
     Wraps ``original_function``, which in most cases is assumed
     to be a `socket.socket` method, to raise an `IOError` for any operations
     on non-local AF_INET sockets.
+
+    Allowing Astropy data also automatically allow GitHub data.
```

Comment:
If this is true, is there really a use case for having a separate option for accessing data from github?

Reply:
@astrofrog said it would be nice to have. Maybe he can comment.